WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -224,10 +228,12 @@ public function is_targetted_token( $stackPtr ) {
 			}
 
 			if ( preg_match( $group['regex'], $token_content ) === 1 ) {
-				$this->process_matched_token( $stackPtr, $groupName, $token_content );
+				$skip_to = $this->process_matched_token( $stackPtr, $groupName, $token_content );
```

Comment:
> Should the highest or the lowest value be returned though ? A case can be made for either one.

Yeah, that's why I say different sniffs might want different behaviors here. But the reason I suggested lowest is because generally skipping isn't a requirement to the proper operation of the sniff, the worst that could happen is an error gets reported twice or something. Whereas if you skip extra beyond a particular check, the sniff might not end up reporting errors that it is supposed to.

But all things considered, I guess it really isn't important to resolve this right now. Maybe it would be best to just leave it as it is, and change the behavior only when we have an actual use-case that requires a change.